src/main/core-options.js

```diff
@@ -62,6 +62,34 @@ const options = {
     `,
     cliCategory: CATEGORY_EDITOR
   },
+  eol: {
+    since: "1.15.0",
+    category: CATEGORY_GLOBAL,
+    type: "choice",
+    default: "auto",
+    description: "Which end of line characters to apply.",
+    exception: value => typeof value === "string",
+    choices: [
+      {
+        value: "auto",
+        description: dedent`
+          Maintain existing
+          (mixed values within one file are normalised by looking at what's used after the first line)
```

Comment:
I'd probably keep it if that does not harm. Otherwise, it is not clear if Prettier keeps mixed line endings or combs them out. Mixed line endings can end up in a file if a text editor is not smart enough.